src/ui/public/stringify/types/url.js

```diff
@@ -93,7 +93,7 @@ export function stringifyUrl(Private) {
 
       switch (this.param('type')) {
         case 'img':
-          return '<img src="' + url + '" alt="' + label + '" title="' + label + '">';
+          return `<img src="${url}" alt="A dynamically-specified image located at ${label}">`;
```

Comment:
> alt="A dynamically-specified image located at ${label}"

Are you sure `label` is the field that you want to display here? If I add a field formatter to the field, and specify a label, then all of the `alt` values are going to be identical. Did you want this to be `url`?